noisy.py

```diff
@@ -249,6 +250,10 @@ def crawl(self):
             except self.CrawlerTimedOut:
                 logging.info("Timeout has exceeded, exiting")
                 return
+            
+            except:
```

Comment:
I'm not sure about this, maybe a better approach would be a logging each time a root URL is selected?

Reply:
The idea is to log with a high criticality level the URL leading to an unrecoverable error.

I am not able to confirm at the moment, but IIRC info log level logs every visited URLs, which might be too verbose when running noisy continuously as a service.